test/test_model.py

```diff
@@ -1199,6 +1199,22 @@ def test_exec(self):
         ])
         self.assertEqual(p, 'fake_exec_process')
 
+    def test_send_signal(self):
+        with self.assertRaises(TypeError):
+            self.container.send_signal('SIGHUP')
+
+        self.container.send_signal('SIGHUP', 's1')
+        self.assertEqual(self.pebble.requests, [
+            ('send_signal', 'SIGHUP', ('s1',)),
+        ])
+        self.pebble.requests = []
+
+        self.container.send_signal('SIGHUP', 's1', 's2')
```

Comment:
add test for the signal.SIGHUP case?

Reply:
I don't think it's necessary given that `model.Container.send_signal` is basically a one-line wrapper around `self._pebble.send_signal(sig, service_names)` (which does test this).